README.md

```diff
@@ -6,17 +6,32 @@ repo or every repo in an organisation.
 Here is the command line usage for the tool:
 
 ```
-usage: update-labels-in-github [options] <file>
+update-labels-in-github.js <labels-spec-file> <org>
 
-Options:
-  -o, --org [org]    the name of the org
-  -r, --repo [repo]  the name of the repo in the format "<REPO_OWNER_NAME>/<REPO_NAME>"
-  -h, --help         output usage information
+Update labels for the specified repo to the given specification config file.
+This will skip any archived repos or included repos that don't exist in an org.
 
 The following environment variables must be set:
- - GITHUB_USER - the git user that owns the token specified in GIT_API_TOKEN
- - GITHUB_API_TOKEN - the git API token used to access the GitHub API
+- GITHUB_USER - the git user that owns the token specified in GIT_API_TOKEN
+- GITHUB_API_TOKEN - the git API token used to access the GitHub API
+
 
+Positionals:
+  labels-spec-file  The labels specification file to update the labels to.
+                                                                        [string]
+  org               The name of the org to update. By default, all repos in the
+                    org will be updated. Use the options --include-repos and
+                    --exclude-repos to limit the repos to update.       [string]
+
+Options:
+  --help               Show help                                       [boolean]
+  --version            Show version number                             [boolean]
```

Comment:
Where does it get the version number from?

Reply:
It parses it from the `package.json`. I could increment that to `0.1.0` to indicate a change in version. With semantic versioning, `0.*.*`, APIs are not considered stable.

Reply:
The other option is that I disable the `--version` option.

Reply:
What is it at the moment 0.0? 0.1.0 sounds reasonable so it's still a work-in-progress perhaps.

Reply:
The version number is `0.0.1`.